src/components/ImageCard/index.js

```diff
@@ -116,7 +117,7 @@ const ImageCard = ({
   }
 
   return (
-    <CardWithoutPadding {...props}>
+    <CardWithoutPadding {...cardProps}>
```

Comment:
This seems like a breaking change... any thoughts on how we can mitigate issues that could potentially arise?

Although it's not pretty nor performant, perhaps `<CardWithoutPadding {...props} {...cardProps}>...` would be safer?

Reply:
I've made it a bit prettier, looks like we have no other way.